src/sql/workbench/services/accountManagement/browser/accountDialog.ts

```diff
@@ -218,6 +221,11 @@ export class AccountDialog extends Modal {
 		const noAccountLabel = localize('accountDialog.noAccountLabel', "There is no linked account. Please add an account.");
 		noAccountTitle.innerText = noAccountLabel;
 
+		this._loadingProviderViewContainer = DOM.$('div.loading-provider-view');
```

Comment:
please reuse the loading spinner component we have: src\sql\base\browser\ui\loadingSpinner\loadingSpinner.ts, what you have is not accessible, it won't announce the status changes.

Reply:
you can refer to table designer code that is using this component.